src/analysis/retail/monk/brewmaster/modules/problems/InvokeNiuzao/index.tsx

```diff
@@ -248,59 +249,65 @@ export function InvokeNiuzaoSection({
 }): JSX.Element {
   const efficiency = castEfficiency.getCastEfficiencyForSpellId(
     talents.INVOKE_NIUZAO_THE_BLACK_OX_BREWMASTER_TALENT.id,
-  )!;
+  );
 
   const wooEfficiency = castEfficiency.getCastEfficiencyForSpellId(
-    SPELLS.WEAPONS_OF_ORDER_BUFF_AND_HEAL.id,
+    talents.WEAPONS_OF_ORDER_BREWMASTER_TALENT.id,
   );
 
   return (
     <Section title="Invoke Niuzao, the Black Ox">
       <p>
-        <SpellLink id={talents.INVOKE_NIUZAO_THE_BLACK_OX_BREWMASTER_TALENT.id} /> is one of the most powerful damage
-        cooldowns in the game&mdash;and one of the most dangerous. Using this ability, we can
-        efficiently turn massive amounts of damage taken into massive amounts of damage dealt.
+        <SpellLink id={talents.INVOKE_NIUZAO_THE_BLACK_OX_BREWMASTER_TALENT.id} /> is one of the
+        most powerful damage cooldowns in the game&mdash;and one of the most dangerous. Using this
+        ability, we can efficiently turn massive amounts of damage taken into massive amounts of
+        damage dealt.
       </p>
       <p>
-        The level 58 upgrade to this ability causes <SpellLink id={talents.INVOKE_NIUZAO_THE_BLACK_OX_BREWMASTER_TALENT.id} /> to
-        add 25% of the purified damage to the next <SpellLink id={SPELLS.NIUZAO_STOMP_DAMAGE.id} />{' '}
-        within 6 seconds. Niuzao casts <SpellLink id={SPELLS.NIUZAO_STOMP_DAMAGE.id} /> when he is
-        summoned, and then every 5 seconds after that, for a total of 5
+        The level 58 upgrade to this ability causes{' '}
```

Comment:
Yea, that section needs a pretty big overhaul actually because of the 10.0 changes. Haven't gotten to it yet, since it also isn't clear whether the talent its talking about is going to remain (it is the *only* "X, Rank 2" talent that didn't get a new name in the last pass---which makes me think it might be getting removed)